solutions/ys-MaxIndepSet.mdx

```diff
@@ -0,0 +1,221 @@
+---
+id: ys-MaxIndepSet
+source: YS
+title: Max Indep Set
+author: Shijie Ren
+---
+
+<Spoiler title="Hint 1">
+
+Generate the complement of the given graph.
+
+The complement or inverse of a graph G is a graph H on the same vertices
+such that two distinct vertices of H are adjacent if and only if
+they are **not** adjacent in G.
+
+That is, to generate the complement of a graph,
+one fills in all the missing edges required to form a complete graph,
+and removes all the edges that were previously there.
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+Consider Bitmask DP when $ N \leq 20 $.
```

Comment:
```suggestion
Consider [Bitmask DP](/gold/dp-bitmasks/) when $ N \leq 20 $.
```

Reply:
this problem is already in the gold/dp-bitmasks module and it may not be proper to link to this module again?

Reply:
oh alr